osg_configure/modules/utilities.py

```diff
@@ -72,14 +72,13 @@ def _compose_attribute_file(attributes):
     """Make the contents of an osg attributes file"""
 
     def islist(var):
-        return type(var) is types.ListType
+        return type(var) is list
```

Comment:
Any reason not to do
```suggestion
        return isinstance(var, list)
```

?